packages/@aws-cdk/aws-ecs/lib/log-drivers/splunk-log-driver.ts

```diff
@@ -25,9 +25,16 @@ export enum SplunkLogFormat {
 export interface SplunkLogDriverProps extends BaseLogDriverProps {
   /**
    * Splunk HTTP Event Collector token.
```

Comment:
Can you update this description to say that this will be stored in the [Options ](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecs-taskdefinition-containerdefinitions-logconfiguration.html#cfn-ecs-taskdefinition-containerdefinition-logconfiguration-options) property and will be viewable in plain text in the console? 

And add that at least one of this property or `secretToken` is required.